<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GloxiniaConfigurator</class>
 <widget class="QMainWindow" name="GloxiniaConfigurator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GloxiniaConfigurator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QTreeWidget" name="treeWidget">
      <column>
       <property name="text">
        <string notr="true">1</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <widget class="QListView" name="listView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_sensor"/>
    <addaction name="actionAdd_module"/>
    <addaction name="actionEdit_sensor"/>
    <addaction name="actionEdit_system"/>
   </widget>
   <widget class="QMenu" name="menuSystem">
    <property name="title">
     <string>System</string>
    </property>
    <widget class="QMenu" name="menuPort_selection">
     <property name="title">
      <string>Port selection</string>
     </property>
    </widget>
    <addaction name="actionConnect"/>
    <addaction name="actionUpdate"/>
    <addaction name="actionDisconnect"/>
    <addaction name="actionConfigure"/>
    <addaction name="menuPort_selection"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuSystem"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAdd_sensor">
   <property name="text">
    <string>Add sensor</string>
   </property>
  </action>
  <action name="actionAdd_module">
   <property name="text">
    <string>Add module</string>
   </property>
  </action>
  <action name="actionEdit_sensor">
   <property name="text">
    <string>Edit sensor</string>
   </property>
  </action>
  <action name="actionEdit_system">
   <property name="text">
    <string>Edit system</string>
   </property>
  </action>
  <action name="actionUpdate">
   <property name="text">
    <string>Update</string>
   </property>
  </action>
  <action name="actionConnect">
   <property name="text">
    <string>Connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionConfigure">
   <property name="text">
    <string>Configure</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
